docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Huh, shouldn't this  have been a vector of 2 elements?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Restart &amp; run all? If this is confirmed that'd be <em>very</em> concerning.</p>

Reply:
<p>it is indeed a vector of length 2: https://pymc--5437.org.readthedocs.build/en/5437/learn/examples/dimensionality.html#vector-distributions (notebooks in pymc are executed automatically every time the documentation is built, that includes PR previews)</p>

Reply:
<p>Reran, its length 2, sorry for the alarm</p>